pkg/operators/oci-handler/oci.go

```diff
@@ -56,6 +56,9 @@ func (o *ociHandler) Name() string {
 }
 
 func (o *ociHandler) Init(params *params.Params) error {
+	if params == nil {
+		return fmt.Errorf("global parameters cannot be nil")
+	}
```

Comment:
Why is it needed?

Reply:
This check ensures that global parameters are not nil before processing to avoid runtime errors. If it's redundant, I can remove it.